flixel/text/FlxText.hx

```diff
@@ -762,6 +775,28 @@ class FlxText extends FlxSprite
 		return value;
 	}
 
+	function get_multiLine():Bool
+	{
+		return (textField != null) ? textField.multiline : false;
+	}
+	
+	function set_multiLine(value:Bool):Bool
+	{
+		if (textField == null)
+			return value;
+			
+		if (textField.multiline != value)
+		{
+			textField.multiline = value;
```

Comment:
should add `_regen = true;` here too

Reply:
Changing `multiline` in the OpenFL text field doesn't require a graphic regeneration. The only thing it changes for non-html text is whether or not new lines can be added by the user in an input text field.

`set_multiline()` here doesn't require refreshing the layout: https://github.com/openfl/openfl/blob/2e105419e01d4b34855260253f6136e4de5f182f/src/openfl/text/TextField.hx#L2835